tools/qfvis/include/vis.h

```diff
@@ -177,6 +180,9 @@ typedef struct threaddata_s {
 	set_pool_t  set_pool;
 	int         id;
 	int         winding_id;
+	unsigned    numsteps;
+	unsigned    numtargetchecks;
+	set_t      *scratchbits;
```

Comment:
`scratchbits` is not a particularly good name. While the set is indeed implemented as bits, that's almost too much information. A better name might be something along the lines of `scratch_set`, `spare_set`, `back_mightsee` (along the lines of `back_buffer`, and its use seems to be for `mightsee`).